.github/dependabot.yml

```diff
@@ -0,0 +1,17 @@
+version: 2
+updates:
+  - package-ecosystem: "pip"
+    directory: "/"
+    schedule:
+      interval: weekly
+    labels:
+      - dependencies
```

Comment:
These labels will need to be added to the repo

Reply:
Labels added!